.env.example

```diff
@@ -0,0 +1,4 @@
+KC_CLIENT_SECRET="ZYhToGucoRMwaWHxXyGKsjunXlXV6Uto"
```

Comment:
this is just an example, right? :laughing:

Reply:
yes, it's moved from `docker-compose` file, already included in https://github.com/app-sre/unleash/blob/master/.gitleaks.toml